File: aula_02/php/Carro.php
<?php

class Carro
{
    /** @var string $cor */
    public $cor;
    /** @var int $numeroPortas */
    public $numeroPortas;

    /**
     * Faz o carro buzinar
     * 
     * @return void
     */
    public function buzinar(): void
    {
        echo "O carro de cor {$this->cor} com {$this->numeroPortas} portas buzinou.\n";
    }
}

File: aula_02/c#/Carro.cs
using System;

namespace projeto
{
    class Carro
    {
        public string cor { get; set; }
        public int numeroPortas { get; set; }

        public void buzinar()
        {
            Console.WriteLine("O carro de cor " + this.cor + " com " + Convert.ToString(this.numeroPortas) + " portas buzinou.");
        }
    }
}
